site/docs/how-to-vote.md

```diff
@@ -0,0 +1,75 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Voting on an Iceberg Release
+
+Before a new version of Apache Iceberg is released, a release candidate will be prepared and a vote
+will occur. Announcements will be made on the Iceberg [developer's mailing list](https://iceberg.apache.org/#community/#mailing-lists)
+and will include links to the following:
+
+- A source tarball
+- A signature (.asc)
+- A checksum (.sha512)
+- Key files
+- GitHub change comparison
+
+After downloading the source tarball, signature, checksum, and key files, follow
+these steps to participate in the voting process.
+
+Import the keys.
+```bash
+gpg --import ${PATH_TO_KEY_FILES}
+```
+
+Verify the signature.
+```bash
+gpg --verify apache-iceberg-${VERSION}.tar.gz.asc
+```
+
+Verify the checksum.
+```bash
+sha512sum -c apache-iceberg-${VERSION}.tar.gz.sha512
+```
+**note**: If you're on a mac that does not have coreutils installed, you can use `shasum -a 512` instead.
+
+Untar the archive and change into the source directory.
+```bash
+tar xzf apache-iceberg-${VERSION}.tar.gz
+cd apache-iceberg-${VERSION}
+```
+
+Run RAT checks to validate license headers.
+```bash
+dev/check-license
+```
+
+Build and test the project using Java 8.
+```bash
+./gradlew build
+```
+
+Votes are cast by replying to the release candidate announcement email on the dev mailing list
+with either +1, 0, or -1.
+
+> [ ] +1 Release this as Apache Iceberg ${VERSION}  
+[ ] +0  
+[ ] -1 Do not release this because...  
+
+In addition to your vote, it's customary to specify if your vote is binding or non-binding. Only members
+of the Project Management Committee have formally binding votes. If you're unsure, you can specify that your
+vote is non-binding. To read more about voting in the Apache framework, checkout the
+[Voting](https://www.apache.org/foundation/voting.html) information page on the Apache foundation's website.
```

Comment:
Could you also add a section on adding the candidate's maven repository to downstream builds? We always supply the maven repository location so that people can add the repository to maven or gradle and test downstream projects easily.

Reply:
It's probably also helpful to add a couple helper sections for how to get the candidate running in Flink or Spark. Here's what I use for Spark:

```bash
/home/blue/Apps/spark-3.1.1-bin-hadoop3.2/bin/spark-shell \
    --conf spark.jars.repositories=https://repository.apache.org/content/repositories/orgapacheiceberg-1019/ \
    --packages org.apache.iceberg:iceberg-spark3-runtime:0.12.1 \
    --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions \
    --conf spark.sql.catalog.local=org.apache.iceberg.spark.SparkCatalog \
    --conf spark.sql.catalog.local.type=hadoop \
    --conf spark.sql.catalog.local.warehouse=/home/blue/tmp/hadoop-warehouse \
    --conf spark.sql.catalog.local.default-namespace=default \
    --conf spark.sql.catalog.prodhive=org.apache.iceberg.spark.SparkCatalog \
    --conf spark.sql.catalog.prodhive.type=hive \
    --conf spark.sql.catalog.prodhive.warehouse=/home/blue/tmp/prod-warehouse \
    --conf spark.sql.catalog.prodhive.default-namespace=default \
    --conf spark.sql.defaultCatalog=local
```

Reply:
I added `Verifying with Spark` and `Verifying with Flink`. If I could get a few extra pairs of eyes on them that'd be much appreciated.